final_task/rss_reader/converters.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+import logging
+import os
+
+import requests
+from dominate import document
+from dominate.tags import *
```

Comment:
It is better to avoid wildcard import whenever you can

Reply:
Done